OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/Internal/LitsItems/AppNews/AppNewsViewModel.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Software Name: Orange Design System (iOS)
+// SPDX-FileCopyrightText: Copyright (c) 2021 - 2023 Orange SA
+// SPDX-License-Identifier: MIT
+//
+// This software is distributed under the MIT license.
+//
+
+import Foundation
+import SwiftUI
+
+// ================================
+// MARK: - App News List View Model
+// ================================
+
+final class AppNewsListViewModel: ObservableObject {
+
+    /// The  state containing or not the release descriptions to use based on the locale
+    @Published var releaseDescriptions: LoadingState<[AboutReleaseDescription], AppNewsListViewModel.Error>
+    /// Some cache to prevent to read each time in filesystem the file and to convert the JSON data which will be bigger and bigger
+    private var releaseDescriptionsCache: Cache<String, [AboutReleaseDescription]>
```

Comment:
Cache in viewModel 🤔

Reply:
so we keep in memory the AppNews description in all the life of the application !!
For me it is a mistake...

Reply:
FI decided both to to use this cache because the czched data are not that important and will increase the app storage with no business interests.